tmt/export.py

```diff
@@ -222,13 +222,18 @@ def export_to_nitrate(test):
 
     # Append id of newly created nitrate case to its file
     if new_test_created:
-        fmf_file_path = test.node.sources[-1]
-        echo(style(f"Append test case id into '{fmf_file_path}'.", fg='green'))
         try:
-            with open(fmf_file_path, encoding='utf-8', mode='a+') as fmf_file:
-                fmf_file.write(f"extra-nitrate: {nitrate_case.identifier}\n")
-        except IOError:
-            raise ConvertError("Unable to open '{0}'.".format(fmf_file_path))
+            with test.node as data:
+                data["extra-nitrate"] = nitrate_case.identifier
+        except AttributeError:
+            # FIXME: remove this deprecated code after FMF modification support is released long enough
+            fmf_file_path = test.node.sources[-1]
+            echo(style(f"Append test case id into '{fmf_file_path}'.", fg='green'))
+            try:
+                with open(fmf_file_path, encoding='utf-8', mode='a+') as fmf_file:
+                    fmf_file.write(f"extra-nitrate: {nitrate_case.identifier}\n")
```

Comment:
I guess we should modify the `dict`, right?

```suggestion
                    data['extra-nitrate'] = nitrate_case.identifier
```

Reply:
backward compatibility not issue.